File: stepik-union/src/main/java/org/stepik/core/courseFormat/stepHelpers/NumberQuizHelper.java
package org.stepik.core.courseFormat.stepHelpers;

import com.intellij.openapi.project.Project;
import org.jetbrains.annotations.NotNull;
import org.stepik.core.courseFormat.StepNode;

/**
 * @author meanmail
 */
public class NumberQuizHelper extends QuizHelper {
    public NumberQuizHelper(@NotNull Project project, @NotNull StepNode stepNode) {
        super(project, stepNode);
    }

    @NotNull
    public String getNumber() {
        initStepOptions();
        return reply.getNumber();
    }
}

File: stepik-core/src/main/kotlin/org/stepik/core/courseFormat/stepHelpers/NumberQuizHelper.kt
package org.stepik.core.courseFormat.stepHelpers

import com.intellij.openapi.project.Project
import org.stepik.core.courseFormat.StepNode


class NumberQuizHelper(project: Project, stepNode: StepNode) : QuizHelper(project, stepNode) {

    val number: String
        get() {
            initStepOptions()
            return reply.number
        }
}
